install.sh

```diff
@@ -485,33 +490,45 @@ configure_linux_token() {
 postinstall_check() {
   detect_mondoo
   if [ $MONDOO_INSTALLED = false ]; then
-    red "${MONDOO_PRODUCT_NAME} installation failed (can't find the Mondoo binary)."
+    red "${MONDOO_PRODUCT_NAME} installation failed (can't find the ${MONDOO_BIN} binary)."
     exit 1
   fi
 
   echo "${MONDOO_PRODUCT_NAME} installation completed."
 }
 
 finalize_setup() {
+
+  # If registration token is provided, register client
   configure_token
 
   # Display final message
-  detect_mondoo_registered
-  if [ $MONDOO_IS_REGISTERED = false ]; then
-    purple_bold "\n${MONDOO_PRODUCT_NAME} is ready to go!"
-    echo
-    lightblue_bold "Next you should register ${MONDOO_PRODUCT_NAME} to get access to policies and reports."
-    lightblue_bold "Follow this guide: "
+  purple_bold "\n${MONDOO_PRODUCT_NAME} is ready to go!"
+
+  # Only if installing Mondoo Client, warn user to register. Do not warn open source users.
+  if [ "$MONDOO_PRODUCT_NAME" = "Mondoo Client" ]; then
+    detect_mondoo_registered
+    if [ $MONDOO_IS_REGISTERED = false ]; then
+      echo
+      lightblue_bold "Next you should register ${MONDOO_PRODUCT_NAME} to get access to policies and reports."
+      lightblue_bold "Follow this guide: "
+      echo
+      lightblue_bold "https://mondoo.com/docs/operating_systems/installation/registration/"
+      echo
+    else
+      echo
+      lightblue_bold "Run 'mondoo scan local' to scan this system or learn more in our quick start docs: https://mondoo.com/docs/"
+      echo
+    fi
+  else
     echo
-    lightblue_bold "https://mondoo.com/docs/operating_systems/installation/registration/"
+    lightblue_bold "Run the following command to scan this system:"
     echo
-  else
-    purple_bold "\n${MONDOO_PRODUCT_NAME} is set up and ready to scan!"
+    echo -e "${MONDOO_BINARY} scan local"
```

Comment:
```suggestion
    echo -e "${MONDOO_BINARY} scan"
```

Reply:
This does make sense for cnquery and cnspec.